markov.py

```diff
@@ -0,0 +1,62 @@
+from bs4 import BeautifulSoup
+import requests
+from numpy.random import choice
+
+
+"""
+Function that pulls text from web
+"""
+megalist = []
+markovdict = {}
+# print("in global scope", id(markovdict))
+def words_from_internet(link='http://www.olin.edu/academic-life/student-affairs-resources/student-life/honor-code/', start='The Olin Honor Code Values', stop='Quick Links'):
+    html = BeautifulSoup(requests.get(link).text, 'lxml')
+    startpoint = html.get_text().find(start)
+    stoppoint = html.get_text().find(stop)
+
+    honor = html.get_text()[startpoint:stoppoint]
+    listhonor = str(honor).split()
+
+    # Combing all text into listhonor
+    return megalist.extend(listhonor)
+
+
+"""
+Make dictionary
+"""
+
+
+def markov(megalist):
+    i = 0
+    # print("in markov", id(markovdict))
+    for i in range(len(megalist)-1):
+        if megalist[i] not in markovdict:
+            markovdict[megalist[i]] = []
+        markovdict[megalist[i]].append(megalist[i+1])
+    return markovdict
+
+
+def smushit(markovdict, megalist):
```

Comment:
Could use a bit more by way of documentation. You've cleaned up the script/jumbles into clean functions, but it's not immediately clear what the functions themselves do.